app.json

```diff
@@ -20,18 +20,33 @@
       "infoPlist": {
         "NSLocationAlwaysAndWhenInUseUsageDescription": "App requires geolocation to retrieve the current location in order to be easier to mark the hotspots.",
         "NSLocationAlwaysUsageDescription": "App requires geolocation to retrieve the current location in order to be easier to mark the hotspots.",
-        "NSLocationWhenInUseUsageDescription": "App requires geolocation to retrieve the current location in order to be easier to mark the hotspots."
+        "NSLocationWhenInUseUsageDescription": "App requires geolocation to retrieve the current location in order to be easier to mark the hotspots.",
+        "NSCameraUsageDescription": "App requires access to your phone’s camera.",
```

Comment:
we need more descriptive messages, Apple won't accept generic messages on app review...
"Camera access is required to take photos of cats"  or something similar.